### ISSUE TITLE
Chess: Controls are not labelled

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#238]
* play button is an empty div. Should be a button (another bug on that) with a textNode label or a aria-label="play" for example.
* The Progress bar under the Winner Prediction is an empty Canvas element. This requires more thought on how to fix, but if the chart part is not important, it could add a sub DOM with the progress value. Probably needs a polite live region added to make the screen reader hear the updated value.
